ui/app/src/components/workspaces/UserResourceItem.tsx

```diff
@@ -10,30 +10,34 @@ import { useComponentManager } from '../../hooks/useComponentManager';
 import { ResourceBody } from '../shared/ResourceBody';
 
 interface UserResourceItemProps {
-  userResource?: UserResource
+  userResource?: UserResource;
```

Comment:
`nit` - pick `;` or `,` - or none, doesn't seem to matter which

Reply:
Done